libsodium-bindings/src/LibSodium/Bindings/SealedBoxes.hs

```diff
@@ -0,0 +1,91 @@
+{-# LANGUAGE CApiFFI #-}
+{-# LANGUAGE Trustworthy #-}
+
+-- |
+-- Module: LibSodium.Bindings.SealedBoxes
+-- Description: Direct bindings to the sealed boxes API of Libsodium
+-- License: BSD-3-Clause
+-- Maintainer: The Haskell Cryptography Group
+-- Stability: Stable
+-- Portability: GHC only
+module LibSodium.Bindings.SealedBoxes
+  ( -- * Introduction
+    -- $introduction
+
+    -- * Functions
+    cryptoBoxSeal
+  , cryptoBoxSealOpen
+
+    -- * Constants
+  , cryptoBoxSealbytes
+  ) where
+
+import Foreign (Ptr)
+import Foreign.C (CUChar, CInt(..), CULLong(..), CSize(..))
```

Comment:
Please don't use wildcard imports.

Reply:
???
It just as everywhere in the project: 
- https://github.com/haskell-cryptography/libsodium-bindings/blob/main/libsodium-bindings/src/LibSodium/Bindings/Secretbox.hs#L38
- https://github.com/haskell-cryptography/libsodium-bindings/blob/main/libsodium-bindings/src/LibSodium/Bindings/AEAD.hs#L26
- https://github.com/haskell-cryptography/libsodium-bindings/blob/main/libsodium-bindings/src/LibSodium/Bindings/GenericHashing.hs#L38
- https://github.com/haskell-cryptography/libsodium-bindings/blob/main/libsodium-bindings/src/LibSodium/Bindings/PasswordHashing.hs#L47
and so on

Reply:
@s-and-witch Yep, and I am about to take care of them too :)

Reply:
Fixed

Reply:
These files are fixed by #54, thank you @s-and-witch